django/src/rdwatch/views/satellite_fetching.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from ninja import Query, Router
+from pydantic import UUID4
+
+from django.http import HttpRequest
+
+from rdwatch.models import SatelliteFetching
+
+router = Router()
+logger = logging.getLogger(__name__)
+
+
+@router.get('/running/')
+def satellite_fetching_running(
+    request: HttpRequest, model_runs: list[UUID4] = Query(None)  # noqa: B008
+):
+    running_sites = SatelliteFetching.objects.filter(
+        status=SatelliteFetching.Status.RUNNING
+    )
+    if model_runs is not None and len(model_runs) > 0:
+        running_sites = running_sites.select_related('sites').filter(
+            site__configuration__in=model_runs
+        )
+    running_site_ids = list(running_sites.values_list('site_id', flat=True))
+    return running_site_ids
```

Comment:
```suggestion
router = RouterPaginated()
logger = logging.getLogger(__name__)


@router.get('/running/')
def satellite_fetching_running(
    request: HttpRequest, model_runs: list[UUID4] = Query(None)  # noqa: B008
):
    running_sites = SatelliteFetching.objects.filter(
        status=SatelliteFetching.Status.RUNNING
    )
    if model_runs is not None and len(model_runs) > 0:
        running_sites = running_sites.filter(
            site__configuration__in=model_runs
        )
    running_site_ids = running_sites.values_list('site_id', flat=True)
    return running_site_ids
```

Two things -
1) I don't think the select_related is necessary
2) Not paginating this endpoint + casting the full queryset to a list could result in excessive memory usage if there are thousands of satellite fetchings all at once. I don't think that would ever realistically happened outside of someone maliciously trying to crash the server, but paginating it is simple enough that it seems like it'd be worth it?